public/app/features/transformers/extractFields/fieldExtractors.ts

```diff
@@ -146,7 +165,7 @@ const extAuto: FieldExtractor = {
       for (const parse of parsers) {
         try {
           const r = parse(v);
-          if (r != null) {
+          if (r != null || r !== undefined) {
```

Comment:
?

Reply:
Oops, left this when troubleshooting the auto extractor

Reply:
But why does the parser check for null, when it is only undefined that is allowed to be returned? I assume the definition of success here is not throwing an error?

Reply:
I knew about the nullish coalescing operator, but I had no idea you could do a nullish check by using a non-strict statement. Thanks!